apps/barista-design-system/src/components/ba-decision-graph/ba-decision-graph-node/ba-decision-graph-node-navigation/ba-decision-graph-node-navigation.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'ba-decision-graph-node-nav',
+  templateUrl: 'ba-decision-graph-node-navigation.html',
+  styleUrls: ['ba-decision-graph-node-navigation.scss'],
+})
+export class BaDecisiongraphNodeNavigation {
+  @Input()
+  _showBackButton: boolean;
```

Comment:
please provide a default that is not undefined.

Please remove the underscores on all inputs we don't have that anywhere in barista – same for outputs.